Packs/UnifiVideoNVR/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "UnifiVideo NVR",
+    "description": "This integration connects to UnifiVideo NVR by Ubiquity networks. \nIt takes motion events and fetches them into incidents. \nWith the integration we can get the video recordings and snapshots",
```

Comment:
--
-> Description is too long to fit in a card view. How about we change it like this?
"description": "This integration connects to UnifiVideo by Ubiquity Networks, which fetches motion events as incidents, allowing you to take video recordings and snapshots"

Reply:
Works for me.